Makefile

```diff
@@ -84,4 +84,22 @@ help:
 				helpMessage = ""; \
 			} \
 		}' \
-		$(MAKEFILE_LIST)
\ No newline at end of file
+		$(MAKEFILE_LIST)
+
+## Setup links in virtual env for development
+develop:
```

Comment:
I'm fine with another workflow for environment management but would like to flesh it out a little more.
I personally am not ready to put in the time to switch everything over, so please in the meantime use `pipenv` and we can open an issue to track separately.

Reply:
Looking into this comment, I am now not sure what you mean by "switch everything over".  "pyenv" is used to determine what python environment you are working from.  The "develop" argument to setup.py adds symbolic links inside the current virtual env.  The "--uninstall" argument removes them.  I use pyenv now.  How would I use it differently for development?

Reply:
Take that back.  I use "pyenv" not "pipenv" and that is just to manage different python versions.  Would have to learn pipenv.

Looking at the docs, I don't see a way to develop multiple packages at once.  This would make it difficult to co-develop.  Using the builtin python utility as below, you can coordinate changes involved in several libraries using one venv.  I have 7 libraries being co-developed.  5 for pyats.genie, one for yang.connector, and one for cisco-gnmi-python.

I will take this out of the makefile for now and look deeper into pipenv when I have the chance.

Reply:
Removed it from Makefile.